Source/AtomicWebView/WebTexture2D.cpp

```diff
@@ -392,6 +390,14 @@ void WebTexture2D::SetSize(int width, int height)
 
 #endif
 
+#if defined(ATOMIC_OPENGL) && defined(ATOMIC_PLATFORM_WINDOWS) 
+    //Force width and height to multiples of 2 so that we don't squish pixels
+    //This doesn't seem to make the text completely sharp but it makes it usable for now.
+    width = (width | 2) + 1;
+    height = (height | 2) + 1;
+    texture_->SetFilterMode(TextureFilterMode::FILTER_NEAREST);
+#endif
```

Comment:
I don't have much clue what is causing this (or why), hence the hack-around.
I might come back to it later but it works OK for my own use at the moment so its not top of my fix list any more... I probably won't go further on this for a couple of months if someone else wants to take a look sooner.